server/core/src/main/java/org/infinispan/server/core/admin/AdminOperationsHandler.java

```diff
@@ -41,9 +41,9 @@ public String getName() {
    }
 
    @Override
-   public <T> CompletableFuture<T> runTask(String taskName, TaskContext context, Executor executor) {
+   public <T> CompletableFuture<T> runTask(String taskName, TaskContext context, BlockingHandler blockingHandler) {
```

Comment:
```suggestion
   public <T> CompletionStage<T> runTask(String taskName, TaskContext context, BlockingHandler blockingHandler) {
```

Reply:
Sure. I didn't scan through the various ones to see if they could be updated. I will take a glance through and if they are private change the return type.